scripts/run-swiftlint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Run swiftlint for local builds so issues show up in Xcode. In CI, swiftlint is called externally and so should not run as part of building any targets.
+
+if [[ -n ${CI+x} ]]; then
+    echo "Detected CI environment. `swiftlint` will run in a dedicated workflow."
+    exit 0;
+fi
+
+/opt/homebrew/bin/swiftlint lint --force-exclude
```

Comment:
Why not like this? My swiftlint is located at /usr/local/bin/swiftlint 
```suggestion
swiftlint lint --force-exclude
```

Reply:
Reverted the build phase additions